eng/Version.Details.xml

```diff
@@ -98,14 +98,14 @@
       <Sha>0385265f4d0b6413d64aea0223172366a9b9858c</Sha>
       <SourceBuild RepoName="test-templates" ManagedOnly="true" />
     </Dependency>
-    <Dependency Name="Microsoft.DotNet.Common.ItemTemplates" Version="7.0.100-rc.1.22404.7" CoherentParentDependency="Microsoft.NET.Sdk">
-      <Uri>https://github.com/dotnet/templating</Uri>
-      <Sha>225ce682f0578db2db5644df5e7024276b39785e</Sha>
-    </Dependency>
-    <Dependency Name="Microsoft.TemplateEngine.Cli" Version="7.0.100-rc.1.22404.7" CoherentParentDependency="Microsoft.NET.Sdk">
-      <Uri>https://github.com/dotnet/templating</Uri>
-      <Sha>225ce682f0578db2db5644df5e7024276b39785e</Sha>
-      <SourceBuild RepoName="templating" ManagedOnly="true" />
+    <Dependency Name="Microsoft.DotNet.Common.ItemTemplates" Version="7.0.100-dev">
+      <Uri>https://github.com/dotnet/sdk</Uri>
+      <Sha>215b2c8c993c2a5ca47f5e76d52122853a29c4ce</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.TemplateEngine.Cli" Version="7.0.100-rc.1.22409.4">
+      <Uri>https://github.com/dotnet/sdk</Uri>
+      <Sha>215b2c8c993c2a5ca47f5e76d52122853a29c4ce</Sha>
+      <SourceBuild RepoName="sdk" ManagedOnly="true" />
```

Comment:
This `SourceBuild` element/metadata should be removed.  It can only be declared once per `RepoName` and it already exists at https://github.com/dotnet/installer/pull/14295/files#diff-fb62e94a1d6f29f863e3d0a22aa38269f6cd1d7f03b109dc06e2cbf2548b86d3R113.

Reply:
Thanks for pointing - removed